Shows how to download photos with batching and prepare them for rendering in a browser (#368)
microsoftgraph/msgraph-sdk-javascript: docs/content/Batching.md

};
```

> ### Download multiple profile photos with batching and prepare manipulate these for rendering in a browser 

Reviewer: ```suggestion
### Download multiple profile photos with batching and preprocess these for rendering in a browser 
```

      reader.onerror = reject;
      reader.onload = _ => {
        resolve(reader.result as string);
      };
      reader.readAsDataURL(blob);
    });
  };

  downloadPhotosBatching = async (client: Client) => {
    try {
      let users = ["user1@contoso.com", "user2@contoso.com"]
  
      // create batch request steps for the users specified above
      const batchRequestSteps : BatchRequestStep[] = users.map((userId) => {
        const request : BatchRequestStep = {
          id: userId,
          request: new Request(`/users/${userId}/photo/$value`, {
            method: "GET",
          })
        };
        return request;
      })
  
      // initiate the batchrequest and execute the operation
      const batchRequestContent = new BatchRequestContent(batchRequestSteps);
      const content = await batchRequestContent.getContent();
      const batchResponse = new BatchResponseContent(await client.api("/$batch").post(content));
  
      // example on how to retrieve the base64 representation of the downloaded image for the first user
      const response = batchResponse.getResponseById(users[0]);
      if (response.ok) {
        var data = await response.text();

>         const _binToBlob = await this.b64toBlob((data),'img/jpg');

Reviewer: ```suggestion
        const binToBlob = await this.b64toBlob((data),'img/jpg');
```

        
        // you can associate the base64 output to an src attribute of an <img> HTML tag

>         const base64Result = await this.blobToBase64(_binToBlob);

Reviewer: ```suggestion
        const base64Result = await this.blobToBase64(binToBlob);
```
